### DISCUSSION TITLE
QSound LL promotion to configuration question.

### DISCUSSION THREAD

**Discussion by john-iv** [mamedev/discussions#7] [General]
Hello Vas, was curious your thoughts on promoting the LL Qsound emulation to a machine configuration for easy toggling in the tab menu.  Last I ran it a month or so ago on my i7-12700K I was hitting roughly 300%.  I remember when this went in at the time a concern was the speed if forced to default.  Is the toggle a viable alternative these days to get some more test coverage?  Thanks -

**Comment by cuavas** [comment]
Not really – the only way to make it user-configurable would be to make it a slot device, and that would be very cumbersome.

IIRC there were still some synchronisation issues that could cause sound issues under certain conditions.  I’m not sure if that improved after the sound stream refactoring or if it’s still an issue.
